src/hooks/login/useLoginService.ts

```diff
@@ -103,6 +108,9 @@ export const useLoginService = (config?: OnProviderLoginType['nativeAuth']) => {
   }) => {
     const loginToken = await getNativeAuthLoginToken();
     tokenRef.current = loginToken;
+    if (!loginToken) {
+      return;
```

Comment:
Would be better to log a warning or an error message here

Reply:
this function is not ready. It must be changed by @StanislavSava or @andreigiura